lib/api/index.js

```diff
@@ -117,7 +147,10 @@ app.get('/', (req, res) => {
   res.send({});
 });
 
-app.get('/:installationId', async (req, res) => {
+app.get('/:installationId', [
+  check('installationId').isInt(),
```

Comment:
oh TIL! this is neat!

Reply:
Yeah this seems to be a very standard express module to add in, and I love the safety it brings. Plus if this were a regular public API, users would get back a response about the section of the validation that failed, but our stafftools client and Jira are the only consumers, so that's less helpful except in dev!